src/rws_poco_client.cpp

```diff
@@ -451,6 +462,24 @@ POCOClient::POCOResult POCOClient::webSocketRecieveFrame()
   return result;
 }
 
+void POCOClient::webSocketClose() {
```

Comment:
```suggestion
void POCOClient::webSocketClose()
{
```
Curly bracket on new line to be consistent with the rest of the code base.